.gitignore

```diff
@@ -16,3 +16,6 @@ bin/
 *.test
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+
```

Comment:
Please add kind-logs-*/ as well

Reply:
`kind-logs-*/` are a product of https://github.com/kubernetes-sigs/kind let's defer that until we actually start using it